devops/create_https_wss_endpoints_aws/devnet.json

```diff
@@ -0,0 +1,43 @@
+{
+  "network": "devnet",
+  "domain_name": "b.hmny.io",
+  "num_of_shards": 4,
+  "num_of_regions": 4,
+  "regions": ["us-east-2", "us-west-1"],
+  "region_vpc": ["vpc-2c420a44","vpc-bb770fdc"],
+  "regions_prod": ["us-east-1","us-east-2","us-west-1"],
+  "region_vpc_prod": ["vpc-88c9c2f3","vpc-2c420a44","vpc-bb770fdc"],
+  "subnet_us-east-1": ["subnet-14da445e", "subnet-2c957212", "subnet-2f00cb73", "subnet-7b65f774", "subnet-7da3751a", "subnet-7e5c8950"],
+  "subnet_us-east-2": ["subnet-45031908", "subnet-b38df5db", "subnet-c071c0ba"],
+  "subnet_us-west-1": ["subnet-551f560e", "subnet-5be2df3c"],
+  "subnet_us-west-2": ["subnet-1953d052", "subnet-6a6c1230", "subnet-c33554ba", "subnet-d73bdcfc"],
+  "sg_us-east-1": ["sg-0a9f239987978e0eb"],
+  "sg_us-east-2": ["sg-05c8f480d9a17574c"],
+  "sg_us-west-1": ["sg-0fc2b150b51b313ec"],
+  "sg_us-west-2": ["sg-05a5b29416bd4b7fd"],
+  "regions_code": ["iad", "cmh", "sfo", "pdx"],
+  "us-east-1-exp-0": [],
+  "us-east-1-exp-1": [],
+  "us-east-1-exp-2": [],
+  "us-east-1-exp-3": [],
+  "us-east-2-exp-0": ["18.221.134.70","18.216.152.7"],
+  "us-east-2-exp-1": ["3.15.202.139", "18.188.135.57"],
+  "us-east-2-exp-2": ["18.217.128.178","3.135.212.174"],
+  "us-east-2-exp-3": ["3.133.130.192", "18.189.182.113"],
+  "us-west-1-exp-0": ["13.52.243.58","18.144.99.201"],
+  "us-west-1-exp-1": ["18.144.81.143","54.67.74.138"],
+  "us-west-1-exp-2": ["18.144.4.73","13.57.23.204"],
+  "us-west-1-exp-3": ["54.183.244.215","13.57.253.75"],
+  "us-west-2-exp-0": [],
+  "us-west-2-exp-1": [],
+  "us-west-2-exp-2": [],
+  "us-west-2-exp-3": [],
+  "explorers_0" : ["54.191.139.12", "54.71.27.215"],
+  "explorers_1" : ["54.241.155.180"],
+  "explorers_2" : ["54.91.183.211"],
+  "tg_s0" : ["tg-s0-api-b-https", "tg-s0-api-b-wss"],
+  "tg_s1" : ["tg-s1-api-b-https", "tg-s1-api-b-wss"],
+  "tg_s2" : ["tg-s2-api-b-https", "tg-s2-api-b-wss"],
+  "tg_s3" : ["tg-s3-api-b-https", "tg-s3-api-b-wss"],
+  "ws_endpoints": ["ws.s0.b.hmny.io", "ws.s1.b.hmny.io", "ws.s2.b.hmny.io","ws.s3.b.hmny.io"]
```

Comment:
what's the relation between the end points and the above values like tg, explorer?  this json config file has to be structured, it is flat right now and hard to figure out the relation among those values.